python/extensions.bzl

```diff
@@ -19,23 +19,48 @@ load("@rules_python//python/pip_install:pip_repository.bzl", "locked_requirement
 load("@rules_python//python/pip_install:repositories.bzl", "pip_install_dependencies")
 load("@rules_python//python/pip_install:requirements_parser.bzl", parse_requirements = "parse")
 load("@rules_python//python/private:coverage_deps.bzl", "install_coverage_deps")
+load("@rules_python//python/private:toolchains_repo.bzl", "get_host_os_arch", "get_host_platform")
 
 def _python_impl(module_ctx):
     for mod in module_ctx.modules:
-        for attr in mod.tags.toolchain:
+        for toolchain_attr in mod.tags.toolchain:
             python_register_toolchains(
-                name = attr.name,
-                python_version = attr.python_version,
+                name = toolchain_attr.name,
+                python_version = toolchain_attr.python_version,
                 bzlmod = True,
                 # Toolchain registration in bzlmod is done in MODULE file
                 register_toolchains = False,
-                register_coverage_tool = attr.configure_coverage_tool,
-                ignore_root_user_error = attr.ignore_root_user_error,
+                register_coverage_tool = toolchain_attr.configure_coverage_tool,
+                ignore_root_user_error = toolchain_attr.ignore_root_user_error,
+            )
+        for interpret_attr in mod.tags.host_python_interpreter:
+            _host_hub(
+                name = interpret_attr.name,
```

Comment:
Reuse the name attribute from toolchain_attr.name (i.e move this into the loop above where the toolchain tag is processed). This avoids any issue of the two being out of sync. The two are closely coupled anyways.